gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -326,25 +446,46 @@ def get_combine_faf_resources(
     return combine_faf_pipeline
 
 
+def filter_gene_to_test(ht: hl.Table) -> hl.Table:
+    """
+    Filter to PCSK9 1:55039447-55064852 for testing.
+
+    :param ht: Table with frequency and FAF information.
+    :return: Table with frequency and FAF information of the filtered interval of a gene
+    """
+    return hl.filter_intervals(
+        ht,
+        [hl.parse_locus_interval("chr1:55039447-55064852", reference_genome="GRCh38")],
+    )
+
+
 def main(args):
     """Create combined FAF resource."""
     hl.init(log="/compute_combined_faf.log")
     test = args.test
     overwrite = args.overwrite
-    pops = list(POPS["v3"] & POPS["v4"])
+    pops = POPS["v3"] + POPS["v4"]
```

Comment:
```suggestion
    pops = list(set(POPS["v3"] + POPS["v4"]))
```
The original code assumed POPS[version] returned a set, this creates a list of unique values.